lib/ansible/modules/web_infrastructure/jboss.py

```diff
@@ -167,15 +167,17 @@ def is_deployed(module):
     elif module.params['deployment_strategy'] == 'filesystem':
         return os.path.exists(os.path.join(module.params['deploy_path'], "%s.deployed" % module.params['deployment']))
     else:
-        rc, stdout, stderr = module.run_command([module.params['cli_path'], '--connect', '--commands=deploy-info'])
+        rc, stdout, stderr = cli_run_commands(module, ['deployment-info', '--name=%s' module.params['deployment']])
```

Comment:
thanks. still trying to figure out how to catch things like this when i run tests locally.